shell/key-bindings.fish

```diff
@@ -1,7 +1,9 @@
 # Key bindings
 # ------------
 function fzf_key_bindings
-
+  set -g __FZF_FISH_MAJOR (echo $FISH_VERSION | cut -f1 -d.)
```

Comment:
I'm fine with either actually. I'm not even a fish user 😆 

Maybe fish has some string manipulation functions we can use in this case, so we don't have to start two subprocesses?